kit/src/elements/file/mod.rs

```diff
@@ -75,9 +75,9 @@ pub fn get_aria_label(cx: &Scope<Props>) -> String {
     cx.props.aria_label.clone().unwrap_or_default()
 }
 
-pub fn emit(cx: &Scope<Props>, s: String) {
+pub fn emit(cx: &Scope<Props>, s: String, key_code: Code) {
```

Comment:
this is the right way to extend the `emit` function, though i really don't like them. i think they make it hard to follow what's happening. i think they also shouldn't be `pub`. that's my 2 cents about that. not a blocking request.